task_processing/metrics.py

```diff
@@ -0,0 +1,54 @@
+import mock
+
+try:
+    import yelp_meteorite
+    METRICS_ENABLED = True
+except Exception:
+    METRICS_ENABLED = False
+
+TASKPROC_TASK_LAUNCHED_COUNT = 'taskproc_task_launched_count'
```

Comment:
@sagar8192 sure the metric name itself can have it (rhs), but I'm saying the LHS is redundant. We're in the taskproc package, we don't need constants to be prefixed with taskproc IMO.